src/lib.rs

```diff
@@ -1,12 +1,12 @@
 #![forbid(unsafe_code)]
 
 pub mod client;
-mod client_receive;
 mod client_send;
+#[doc(hidden)] // TODO: move to imap-codec and make it public
```

Comment:
We could create an issue and reference it here.

Reply:
I've created #238